functions/getCrousMenus.js

```diff
@@ -43,6 +43,13 @@ async function fetchMenu(URL) {
     //To remove **** in the sirtaki menu
     plat = plat.replace('****','');
     plat = plat.replace('****','');
+
+    //Fix: "<br>" in Space Campus menu
```

Comment:
Those comments are not necessary. Dont't put a comment each time you fix somethings related to scapping